index.d.ts

```diff
@@ -230,6 +230,23 @@ declare module '@primer/components' {
   export const Sticky: React.FunctionComponent<PositionComponentProps>
   export const Fixed: React.FunctionComponent<PositionComponentProps>
 
+  export interface SideNavProps extends CommonProps, BorderProps, Omit<React.HTMLAttributes<HTMLDivElement>, 'color'> {
+    bordered?: boolean
+    variant?: 'normal' | 'lightweight'
+  }
+
+  export interface SideNavLinkProps extends CommonProps, TypographyProps, Omit<React.HTMLAttributes<HTMLAnchorElement>, 'color'> {
+    href?: string
+    muted?: boolean
+    selected?: boolean
+    underline?: boolean
+    variant: 'normal' | 'full'
```

Comment:
This should be optional
```suggestion
    variant?: 'normal' | 'full'
```